R/check-cran.R

```diff
@@ -9,6 +9,9 @@
 #' * It checks the package on R-release and R-devel.
 #' * It uses the `--as-cran` argument to `R CMD check`.
 #' * It requires all dependencies, including suggested ones.
+#' 
+#' @details This function is wrapped by [devtools::check_rhub()] which you 
+#' might find useful if you load `devtools` via your .Rprofile (see [usethis::use_devtools()]).
```

Comment:
I think to link to these packages, we would need to Suggest them, which is not ideal. So maybe you can just mention the function names without creating actual links?